rotkehlchen/api/v1/schemas.py

```diff
@@ -705,10 +704,26 @@ def make_history_event_filter(
         else:
             filter_query = HistoryEventFilterQuery.make(**common_arguments)
 
-        return {
+        base_filter = {
             'filter_query': filter_query,
-            'group_by_event_ids': data['group_by_event_ids'],
         }
+        base_filter.update(self.get_extra_fields(data))
+        return base_filter
```

Comment:
maybe is cleaner to do

```python
     return self.get_extra_fields(data) | {
            'filter_query': filter_query,
            'group_by_event_ids': data['group_by_event_ids'],
        }
```

Reply:
yes, it's cleaner

re: group_by_event_ids
I don't want it in the second schema, so it's still in the extra_fields